### PR TITLE
[Redesign] Address footer, SVG issues, and package details issues

### PR DESCRIPTION
**Pull request by joelverhagen**
Three classes of issues fixed here:
1. Footer issues
1. SVG issues
1. Default icon issues

A couple of the issues were interconnects so I though this would be easier to review than 10 different PRs.

- https://github.com/NuGet/NuGetGallery/issues/4137 - NuGet logo is cut off a bit at the bottom
- https://github.com/NuGet/NuGetGallery/issues/4139 - .NET foundation icon in footer should be SVG
- https://github.com/NuGet/NuGetGallery/issues/4152 - White bar appears at the bottom of the page overlapping text in Edge
- https://github.com/NuGet/NuGetGallery/issues/4159 - Define a fallback font
- https://github.com/NuGet/NuGetGallery/issues/4165 - .NET Foundation logo jumps to the right in small window in Edge
- https://github.com/NuGet/NuGetGallery/issues/4166 - Pages have a lot of empty space at the bottom in Edge at certain window width
- https://github.com/NuGet/NuGetGallery/issues/4167 - Narrow screens cause white block under footer
- https://github.com/NuGet/NuGetGallery/issues/4169 - Package icon should be centered and consistent size on package details
- https://github.com/NuGet/NuGetGallery/issues/4182 - Facebook, twitter icons should be SVG
- https://github.com/NuGet/NuGetGallery/issues/4183 - White borders around icons in high contrast in edge

### CODE REVIEW

**Comment by joelverhagen** [comment]
As usual, it's less interesting to review the CSS. Instead, take a look at the LESS.

**Comment by joelverhagen** [comment]
🔔

File: src/NuGetGallery/Views/Pages/Home.cshtml [hunk 1]
```diff
@@ -50,23 +50,29 @@
...
+                     @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/orange-circle-225x225.png")) />
                 <h3>Upload</h3>
                 <p>
                     Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed porttitor, neque id consectetur fringilla, odio
                     justo consectetur leo, nec interdum odio odio non lacus. Donec commodo enim et suscipit ultrices.
                 </p>
             </div>
             <div class="col-sm-4">
-                <img src="~/Content/gallery/img/download-hero.png" alt="Cartoon image a toolbox" />
+                <img width="225" height="225" alt="Cartoon image a toolbox"
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: `Cartoon image a toolbox` -> `Cartoon image of a toolbox`

File: src/NuGetGallery/Views/Shared/Gallery/Footer.cshtml [hunk 2]
```diff
@@ -24,30 +24,29 @@
                 </p>
             </div>
         </div>
-        <div class="row row-gap">
-            <div class="col-sm-3">
-                <img alt="" aria-hidden="true" width="184" height="53"
+        <div class="row">
+            <div class="col-md-3 row-gap">
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Shared/Gallery/Footer.cshtml]: This change (`sm` -> `md`) will make the footer expand into single rows much sooner, correct? Do we want that? The sizing seems fine to me right now IMO.

Comment (joelverhagen) [inline src/NuGetGallery/Views/Shared/Gallery/Footer.cshtml]: Yes, this is intentional. `sm` is small enough that it causes some ugly line wrapper.

File: src/NuGetGallery/Views/Pages/Home.cshtml (hunk 1 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: Fixed.

**Pull request merged**